db/migrations/mysql/000102_add_sentat_to_notifyadmin.up.sql

```diff
@@ -0,0 +1,14 @@
+SET @preparedStatement = (SELECT IF(
+    NOT EXISTS(
+        SELECT 1 FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'NotifyAdmin'
+        AND table_schema = DATABASE()
+        AND column_name = 'SentAt'
+    ),
+    'ALTER TABLE NotifyAdmin ADD COLUMN SentAt bigint;',
```

Comment:
nit: wondering if we should use `DEFAULT NULL` here, instead of inserting new lines with a value hardcoded to 0?